### DISCUSSION TITLE
Is langflow production ready?

### DISCUSSION THREAD

**Discussion by ronaldo-moco** [langflow-ai/langflow#1516] [General]
Considering the version number, is langflow in alpha or beta?
Is it ready for production? 
Do we have any use case that proves the capabilities of langflow in a real scenario?

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
